docs/dev/table/tableApi.md

```diff
@@ -373,7 +373,7 @@ Table result = orders
     <tr>
       <td>
         <strong>Distinct</strong><br>
-        <span class="label label-primary">Batch</span>
+        <span class="label label-primary">Batch</span> <span class="label label-primary">Streaming</span>
```

Comment:
If the input is append/insert only, distinct will only produce insert changes as well. 
However, I don't think we make this distinction in the decoration phase. AFAIK, it will always be marked to produce updates right now.

IMO, it would be good if DISTINCT on append-only input would not be treated as producing updates. We don't have to fix it in this PR, but I think it would be good to make such a change. What do you think @wuchong?